rfcs/0005-server-middleware-extensibility.md

```diff
@@ -0,0 +1,154 @@
+- Start Date: 2019-04-23
+- RFC PR: -
+- Issue: -
```

Comment:
`https://github.com/SAP/ui5-server/issues/113` (already forgot about this, sorry 😅)